lib/layers/domain/entities/achieves.dart

```diff
@@ -0,0 +1,45 @@
+import '../../data/models/remote/achievements_data/achievement_data.dart';
```

Comment:
Using model from data layer like `AchievementData` broke the clean architecture because domain should not know anything from the data layer.